Extensions/News/Compiler.js

```diff
@@ -1,3 +1,6 @@
+/** News object which was intended to parse each section from earlier downloaded Edupage News into usable texts
```

Comment:
why don't we just delete everything that is deprecated? It will still be in git history in one way or another, so why keep it in code?

Reply:
It's done in the other PR